src/client/componentsMealsharing/PostForm.js

```diff
@@ -0,0 +1,85 @@
+import React,{Component,useState} from "react";
+
+export function PostForm() {
+    const [seat, setSeat] = useState("");
+    const [phone, setPhone] = useState("");
+    const [name, setName] = useState("");
+    const [mail, setMail] = useState("");
+    const [reservation, setReservation] = useState([]);
+    bookSeats = ()=> {
```

Comment:
```suggestion
    const bookSeats = ()=> {
```

Reply:
sorry, I did not use Postform component here. I moved it to reservationSpecific component.